jest.config.js

```diff
@@ -4,4 +4,5 @@ module.exports = {
   },
   moduleFileExtensions: ['ts', 'tsx', 'js', 'jsx', 'json'],
   testEnvironment: 'node',
+  rootDir: './test',
```

Comment:
Jest would execute the .spec files in the directory `test-lib-consumption` without that.